include/RAJA/policy/cuda/kernel/CudaKernel.hpp

```diff
@@ -136,7 +136,17 @@ using CudaKernelOccAsync =
  */
 template <size_t num_threads, typename... EnclosedStmts>
 using CudaKernelFixed =
-    CudaKernelExt<cuda_explicit_launch<false, operators::limits<size_t>::max(), num_threads>,
+    CudaKernelExt<cuda_explicit_launch<false, operators::limits<size_t>::max(), 1, num_threads>,
+                  EnclosedStmts...>;
+
+/*!
+ * A RAJA::kernel statement that launches a CUDA kernel with a fixed
+ * number of threads (specified by num_threads) and min blocks per sm.
+ * The kernel launch is synchronous.
+ */
+template <size_t num_threads, size_t blocks_per_sm, typename... EnclosedStmts>
```

Comment:
Order of blocks_per_sm and num_threads should be consistent

Reply:
What is the order you'd like for `cuda_launch`? Then I'll make everything else follow. Currently it's
```c++
template <bool async0, size_t num_blocks, size_t blocks_per_sm, size_t num_threads>
struct cuda_launch {};
```

Reply:
My first reaction is to put `blocks_per_sm` at the end so existing code just has to add a number at the end. But I can also see the logic of putting `blocks_per_sm` between `num_blocks` and `num_threads`. Does anyone else have an opinion?

Reply:
Done, I think I like that better to.

Reply:
I agree with the first reaction of @MrBurmark